www/source/partials/docs/_glossary-packages.html.md.erb

```diff
@@ -1,33 +1,37 @@
-# <a name="glossary-packages" id="glossary-packages" data-magellan-target="glossary-packages">Packages</a>
+# <a name="glossary-artifacts" id="glossary-artifacts" data-magellan-target="glossary-artifacts">Packages</a>
 
-A package refers to a binary distribution for a given piece of software built with Habitat that contains a software library or application and any configuration information for that software. It's a signed tarball with a .hart file extension created from a plan definition and built with Habitat tools that can be post-processed to a specific format, such as when creating a Docker container.
+A package refers to a binary distribution for a given piece of software built with Habitat. A Habitat artifact is a signed tarball with a `.hart` file extension, and is sometimes referred to as a "heart", or "hartifact". Habitat artifacts are composed of a software library or application, the configuration information for that software, and lifecycle hooks. They are created from a the plan file, a `plan.sh` on Linux systems or a `plan.ps1` on Windows systems, and are built with Habitat tools. Habitat artifacts can be exported to a specific format, such as when creating a Docker container.
```

Comment:
This appears to be the same as www/source/partials/docs/_glossary-artifacts.html.md.erb, so the same comments apply.